Detectors/TPC/base/include/TPCBase/CDBInterface.h

```diff
@@ -119,6 +119,12 @@ class CDBInterface
   /// \return gain map object
   const CalPad& getGainMap();
 
+  /// Return any CalPad object object
+  ///
+  /// The function returns the CalDet object stored at the given path in the CCDB
+  /// \return CalPad object
+  const CalPad& getAnyObject(const std::string_view path);
```

Comment:
I think the name is misleading. Sind `getAnyObject` hints that any kind of object can be extracted. But it specifically extracts `CalPad` objects. So perhaps call it `getCalPad`

Reply:
Good point, I changed it